.github/workflows/build.yml

```diff
@@ -59,6 +55,27 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-go-
 
+    - name: Test
+      run: make test
+      working-directory: amazon-cloudwatch-agent
+
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v2
+      with:
+        verbose: true
+        directory: ./amazon-cloudwatch-agent/
+
     - name: Build
-      run: make test build
+      run: make build
       working-directory: amazon-cloudwatch-agent
+
+    - name: Archive production artifacts
```

Comment:
Is there anything we should add to the README/docs about the artifacts that we are archiving?

Reply:
I think I'm going to drop the archive, we already do the nightly builds.  Not really needed right now.